src/SFA.DAS.AANHub.Domain/Common/MemberRemovalStatus.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace SFA.DAS.AANHub.Domain.Common;
```

Comment:
Create a proper MemberStatus enum instead, which also has option of Active

Reply:
I will leave this as only Deleted or Removed for now as 'active' will have its own logical path to be done.  But I will rename to MemberStatus.cs